src/Uno.Playground.Shared/Styles/StyleGuide/Colors.xaml

```diff
@@ -4,32 +4,45 @@
 
 	<!-- Color Resources -->
 	<ResourceDictionary.ThemeDictionaries>
-		<ResourceDictionary x:Key="Default">
-			<Color x:Key="Color01">#FFFFFFFF</Color>
-			<Color x:Key="Color02">#FF000000</Color>
-			<Color x:Key="Color03">#FF159BFF</Color>
-			<Color x:Key="Color04">#FF242424</Color>
-			<Color x:Key="Color05">#80242424</Color>
-			<Color x:Key="Color06">#FF313131</Color>
-			<Color x:Key="Color07">#FF7A67F8</Color>
-			<Color x:Key="Color08">#FF67E5AD</Color>
-			<Color x:Key="Color09">#FFCED0D3</Color>
-			<Color x:Key="Color10">#FF919191</Color>
-			<Color x:Key="Color11">#FFF85977</Color>
-			<Color x:Key="Color12">#FF9B9B9B</Color>
-			<Color x:Key="Color13">#FF229DFC</Color>
-			<Color x:Key="Color14">#FFF65678</Color>
-			<Color x:Key="Color15">#FF6CE5AE</Color>
-			<Color x:Key="Color16">#FF7A69F5</Color>
-			<Color x:Key="Color17">#FFD8D8D8</Color>
-			<Color x:Key="Color18">#1A159BFF</Color>
-			<Color x:Key="Color19">#FF272C35</Color>
-			<Color x:Key="Color20">#FFEEEEEE</Color>
-			<Color x:Key="Color21">#FF7A7A7A</Color>
-			<Color x:Key="Color22">#FFE6E6E6</Color>
+		<ResourceDictionary x:Key="Light">
+			<Color x:Key="AndroidBlue">#2196F3</Color>
```

Comment:
We generally avoid naming keys with their actual color name, this avoids confusion when changed later on. Is it `AndroidForeground` instead? /cc @agneszitte-nventive

Reply:
@agneszitte I agree we shouldn't have these styles in Playground.

In this case we will need to update the sample itself (I don't know where those samples are maintained)

Reply:
- +1 for color-name as name; you should describe "what it is used for"(purpose) than "what it is"(color).
  use this syntax: {Qualifier}{Control}{Resource}{State?}
  eg: AndroidButtonBackground, IosButtonBackgroundDisabled // pick your poison: Ios or iOS
- I would keep them inside DefaultStyles (which houses the control style), those colors are closely related to the specific control style.
- It is, more or less, fine that `Color\d{2}` are not theme-dependent. However, I would fear that some shared samples could be relying on these resources to be specific color. I would be hesitant to touch them now.

Reply:
on the bright side, the control styles in `DefaultStyles` or `Styles\Controls\*` all seem to be x:Key-ed
so changing them should be less of a problem

Reply:
@Youssef1313 is there something to adjust here?

Reply:
On a second thought, I think we should really remove all these styles. The expectation should be that code in Playground matches behavior when copied into an Uno Wasm app right? But that is not the case with Playground defining own styles and overriding some control templates, etc.

The real fix would be to remove all these and also update the Playground snippets (needs someone with access to the API).

I'm not sure though if we can merge this PR until we have the proper fix (assuming you agree that it's the proper fix)

Reply:
When updating the snippets, we also don't want to include any explicit colors because that will work on either light or dark but likely not both.